addons/mail_bot/static/src/js/notification_alert.js

```diff
@@ -3,20 +3,56 @@ odoo.define('mail.NotificationAlert', function (require) {
 
 var Widget = require('web.Widget');
 var widgetRegistry = require('web.widget_registry');
+var core = require('web.core');
 
+var _t = core._t;
 // -----------------------------------------------------------------------------
 // Display Notification alert on user preferences form view
 // -----------------------------------------------------------------------------
 var NotificationAlert = Widget.extend({
-   template: 'mail.NotificationAlert',
-   /**
-    * @override
-    */
-   init: function () {
-      this._super.apply(this, arguments);
-      var hasRequest = this.call('mailbot_service', 'isRequestingForNativeNotifications');
-      this.isNotificationBlocked = window.Notification && window.Notification.permission !== "granted" && !hasRequest;
-   },
+    template: "mail.NotificationAlert",
```

Comment:
No diff here, keep single quotes.

Technical string have single quotes, while UI strings have double quotes. I will comment each that should be changed.